### PR TITLE
Introduced KDCustomCheckbox component and fixed KDInputView::setValue.

### PR DESCRIPTION
**Pull request by fatihacet**

### CODE REVIEW

File: lib/styles/default/kd.input.styl [hunk 1]
```diff
@@ -417,3 +417,80 @@ KDInputView > KDTokenizedInputView styles
...
+    width                   0
+
+  25%
+    height                  0
+    width                   5px
+
+  50%
+    height                  10px
+    width                   5px
+
```
Comment (fatihacet) [inline lib/styles/default/kd.input.styl]: @sinan do we also need to add `@-moz-keyframe` and `@-webkit-keyframes` or `@keyframes` will be enough?

File: lib/components/inputs/inputview.coffee [hunk 2]
```diff
@@ -235,15 +235,22 @@ module.exports = class KDInputView extends KDView
...
+    if @getOption('type') in [ 'checkbox', 'radio' ]
       if value
-      then el.setAttribute "checked", "checked"
-      else el.removeAttribute "checked"
+        el.setAttribute 'checked', 'checked'
+        el.checked = yes
+      else
+        el.removeAttribute 'checked'
+        el.checked = no
+
```
Comment (fatihacet) [inline lib/components/inputs/inputview.coffee]: I spend half an hour see what's going on here and fix it. The problem was if the input is checked by clicking the DOM element then `setValue no` wasn't working.

File: lib/styles/default/kd.input.styl (hunk 1 above)
Comment (gokmen) [inline lib/styles/default/kd.input.styl]: you need to do it for `@-webkit-keyframes` and `@-moz-keyframe` `@keyframes` not necessary afterwards, had good times with them in here https://github.com/koding/kd/blob/master/src/themes/default/kd.slide.styl :)

File: lib/components/inputs/inputview.coffee (hunk 2 above)
Comment (gokmen) [inline lib/components/inputs/inputview.coffee]: good catch :+1:

**Comment by gokmen** [comment]
LGTM :+1:

**Pull request merged**